data_processors/pipeline/lambdas/bcl_convert.py

```diff
@@ -340,7 +343,10 @@ def handler(event, context) -> dict:
     sqr = sequence_run_srv.get_sequence_run_by_run_id(seq_run_id) if seq_run_id else None
 
     # construct and format workflow run name convention
-    workflow_run_name = wfl_helper.construct_workflow_name(seq_name=seq_name, seq_run_id=seq_run_id)
+    workflow_run_name = wfl_helper.construct_workflow_name(
+        seq_name=seq_name,
+        seq_run_id=seq_run_id,
+        portal_uuid=portal_run_uuid)
```

Comment:
ditto consistent `portal_run_id` naming; instead of `portal_uuid` or `portal_run_uuid`